examples/nidaqmx_analog_input/NIDAQmxAnalogInput.serviceconfig

```diff
@@ -5,7 +5,12 @@
       "serviceClass": "ni.examples.NIDAQmxAnalogInput_Python",
       "descriptionUrl": "",
       "providedInterfaces": [ "ni.measurementlink.measurement.v1.MeasurementService" ],
-      "path": "start.bat"
+      "path": "start.bat",
+      "annotations": {
+        "ni/service.description": "Measurement example that uses NI-DAQmx to measure analog input.",
```

Comment:
These descriptions don't match the `pyproject.toml` files. 

- "MeasurementLink example" vs "Measurement example"
- "that performs a finite analog input measurement with NI-DAQmx" with "that uses NI-DAQmx to measure analog input"
- "that sources and measures a DC voltage with an NI SMU" vs. "that interacts with an NI-DCPower instrument"
- etc.

Some of the paraphrased descriptions have grammar errors, such as "a NI-Digital". It should use "an" because "NI" is pronounced "enn eye".

"NI-DMM" is a driver software package.  "An NI DMM" is a DMM instrument made by NI.

"NI-FGEN" is a driver software package. "An NI waveform generator" is an instrument made by NI.

There is no such thing as an "NI-Digital SPI". This example uses an NI-Digital instrument to test a DUT that supports SPI.

Reply:
Never so the description was on pyproject. Will fix